CHANGELOG.md

```diff
@@ -1,5 +1,195 @@
 # Changelog
 
+## 🎁 1.3.0
+
+- ### Features
+
+  - **Environments - [EverlastingBugstopper], [issue/385][pull/386]**
+
+    Wrangler 1.3.0 includes supports for **environments**, allowing developers to deploy Workers projects to multiple places. For instance, an application can be deployed to a production URL _and_ a staging URL, without having to juggle multiple configuration files.
+
+    To use environments, you can now pass in `[env.$env_name]` properties in your `wrangler.toml`. Here's an example:
+
+    ```toml
+    type = "webpack"
+    name = "my-worker-dev"
+    account_id = "12345678901234567890"
+    zone_id = "09876543210987654321"
+    workers_dot_dev = false
+
+    [env.staging]
+    name = "my-worker-staging"
+    route = "staging.example.com/*"
+
+    [env.production]
+    name = "my-worker"
+    route = "example.com/*"
+    ```
+
+    With multiple environments defined, `wrangler build`, `wrangler preview`, and `wrangler publish` now accept a `--env` flag to indicate what environment you'd like to use, for instance, `wrangler publish --env production`.
+
+    To support developers transitioning to environments, we've written documentation for the feature, including further information about deprecations and advanced usage. [Check out the documentation here!](https://github.com/cloudflare/wrangler/blob/master/docs/environments.md)
+
+    [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+    [issue/385]: https://github.com/cloudflare/wrangler/issues/385
+    [pull/386]: https://github.com/cloudflare/wrangler/pull/386
+
+  - **KV commands - [ashleymichal], [issue/339][pull/405]**
```

Comment:
if we can list more than one contributor, i think we should put gabbi here, she wrote A LOT of this code.
```suggestion
  - **KV commands - [ashleymichal, gabbifish], [issue/339][pull/405]**
```

Reply:
ah totally! my little keyboard shortcut only grabs the PR creator. good call!